tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/api/OpenPaasRestClient.java

```diff
@@ -35,6 +37,8 @@ public OpenPaasRestClient(OpenPaasConfiguration openPaasConfiguration) {
 
         this.client = HttpClient.create()
             .baseUrl(apiUrl.toString())
+            .secure(sslContextSpec -> sslContextSpec.sslContext(
+                SslContextBuilder.forClient().trustManager(InsecureTrustManagerFactory.INSTANCE)))
```

Comment:
likely we should have an option to trust all SSL certs here.

And default to `false`.

Reply:
probably should have an all optional ssl conf